big_o_exercise/readme.md

```diff
@@ -5,16 +5,47 @@
 Simplify the following big O expressions as much as possible:
 
 1. `O(n + 10)`
+
+		O(n)
+
+
 2. `O(100 * n)`
+
+		O(n)
+
+
 3. `O(25)`
+
+		O(1)
+		
 4. `O(n^2 + n^3)`
+
+		O(n^3)
+		
 5. `O(n + n + n + n)`
+
+		O(n)
+		
 6. `O(1000 * log(n) + n)`
+
+		O(n)
+		
 7. `O(1000 * n * log(n) + n)`
+
+		O(n)
+		
 8. `O(2^n + n^2)`
+
+		O(n^2)
```

Comment:
double check this one too (i recognize this one is a bit challenging, since we didn't explicitly talk about exponentials vs. quadratics in class)